### PR TITLE
i18n(zh-tw): update `layouts.mdx` translation

### PR DESCRIPTION
**Pull request by mingjunlu**
#### Description

- Updates `src/content/docs/zh-tw/basics/layouts.mdx`

#### Related issues & labels

- ~~Closes #~~
- Suggested label: i18n

### CODE REVIEW

**Review by liruifengv — APPROVED** [review]
LGTM. It looks great. Thanks @mingjunlu

**Comment by liruifengv** [comment]
As there are still some differences between Traditional Chinese and Simplified Chinese. So I'll wait for the other maintainer to look at it before merging it.

**Comment by liruifengv** [comment]
Hey @dreyfus92 Cloud you help to double-check this PR?

File: src/content/docs/zh-tw/basics/layouts.mdx
Comment (dreyfus92) [inline src/content/docs/zh-tw/basics/layouts.mdx:192]:
```suggestion
* 標題資訊（即 `h1 -> h6` 元素）透過 `headings` 陣列存取，而非 `getHeadings()` 函式。

* `file` 和 `url` *也*可在巢狀 `frontmatter` 屬性下存取（即 `frontmatter.url` 和 `frontmatter.file`）。

* 無法存取在 frontmatter 以外宣告的數值（例如 MDX 的 `export` 表達式）。如有類似需求，請考慮[匯入版面](#手動匯入版面mdx)。
```

File: src/content/docs/zh-tw/basics/layouts.mdx
Comment (dreyfus92) [inline src/content/docs/zh-tw/basics/layouts.mdx:225]: This comment can be translated.

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @mingjunlu, I left you a couple of small suggestions 😁

File: src/content/docs/zh-tw/basics/layouts.mdx
Comment (mingjunlu) [reply src/content/docs/zh-tw/basics/layouts.mdx]: Thanks for spotting this! Comment translated: fe96551219952e3c15397b90d818c40684369dcb

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job 🚀

**Pull request merged**
